### PR TITLE
feat: making all secret management across charts consistent

### PR DESCRIPTION
**Pull request by ToonvanStrijp**
## Related Issue or Design Document
#260 

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

**Comment by ToonvanStrijp** [comment]
@Demonsthere third time's a charm :100:

File: helm/charts/hydra/values.yaml [hunk 1]
```diff
@@ -55,6 +55,19 @@ service:
     #      lines, adjust them as necessary, and remove the curly braces after 'labels:'.
     #      e.g.  app: hydra
 
+secret:
+  # -- switch to false to prevent creating the secret
+  enabled: true
+  # ... and choose a different name for a secret you provide like this
+  # nameOverride: "MyOtherName"
```
Comment (Demonsthere) [inline helm/charts/hydra/values.yaml:62]:
```suggestion
  # -- Provide custom name of existing secret, or custom name of secret to be created
  nameOverride: ""
  # nameOverride: "myCustomSecret"
```

File: helm/charts/hydra/values.yaml [hunk 2]
```diff
@@ -55,6 +55,19 @@ service:
...
+  secretAnnotations:
```
Comment (Demonsthere) [inline helm/charts/hydra/values.yaml:63]:
```suggestion
  # -- Annotations to be added to secret. Annotations are added only when secret is being created. Existing secret will not be modified.,
  secretAnnotations:
```

File: helm/charts/oathkeeper/values.yaml [hunk 3]
```diff
@@ -141,14 +141,19 @@ oathkeeper:
...
-  # -- set to false for this helm chart to NOT manage the secret
-  # -- defaults to true
-  manage: true
-
-  # -- name of the secret to use. If empty, defaults to {{ include "oathkeeper.fullname" . }}
-  name: ""
+  # -- switch to false to prevent creating the secret
+  enabled: true
 
+  # ... and choose a different name for a secret you provide like this
```
Comment (Demonsthere) [inline helm/charts/oathkeeper/values.yaml:147]: Please keep the comments consistent between charts ;) I have added suggestions in the first one

**Review by Demonsthere — CHANGES_REQUESTED** [review]
Hello there!
Just adding some finishing touches, but otherwise LGTM :)

**Comment by ToonvanStrijp** [comment]
@Demonsthere done :heavy_check_mark:

**Review by Demonsthere — CHANGES_REQUESTED** [review]
I think you missed something in oathkeeper ;) 
- https://github.com/ory/k8s/blob/master/helm/charts/oathkeeper/templates/deployment-controller.yaml#L57
- https://github.com/ory/k8s/blob/master/helm/charts/oathkeeper/templates/deployment-sidecar.yaml#L45
-

**Review by Demonsthere — APPROVED** [review]
All fine now! :)

**Pull request merged**